$MirOS: ports/x11/windowmaker/patches/patch-WPrefs_app_Menu_c,v 1.1.7.1 2005/03/18 16:00:58 tg Exp $
--- WPrefs.app/Menu.c.orig	Tue Oct 26 02:23:17 2004
+++ WPrefs.app/Menu.c	Wed Apr 26 12:37:03 2006
@@ -577,7 +577,7 @@ createPanel(_Panel *p)
         data->param.exec.command = "xv";
 
         data = putNewItem(panel, pad, ExecInfo, _("Acrobat Reader"));
-        data->param.exec.command = "acroread || /usr/local/Acrobat4/bin/acroread";
+        data->param.exec.command = "acroread";
 
         data = putNewItem(panel, pad, ExecInfo, _("ghostview"));
         data->param.exec.command = "gv";
@@ -616,17 +616,17 @@ createPanel(_Panel *p)
 
         data = putNewItem(panel, pad, DirectoryInfo, _("Themes"));
         data->param.directory.command = "setstyle";
-        data->param.directory.directory = "/usr/share/WindowMaker/Themes /usr/local/share/WindowMaker/Themes $HOME/GNUstep/Library/WindowMaker/Themes";
+        data->param.directory.directory = "/usr/mpkg/share/WindowMaker/Themes $HOME/GNUstep/Library/WindowMaker/Themes";
         data->param.directory.stripExt = 1;
 
         data = putNewItem(panel, pad, DirectoryInfo, _("Bg Images (scale)"));
         data->param.directory.command = "wmsetbg -u -s";
-        data->param.directory.directory = "/opt/kde2/share/wallpapers /usr/share/WindowMaker/Backgrounds $HOME/GNUstep/Library/WindowMaker/Backgrounds";
+        data->param.directory.directory = "/usr/mpkg/share/WindowMaker/Backgrounds $HOME/GNUstep/Library/WindowMaker/Backgrounds";
         data->param.directory.stripExt = 1;
 
         data = putNewItem(panel, pad, DirectoryInfo, _("Bg Images (tile)"));
         data->param.directory.command = "wmsetbg -u -t";
-        data->param.directory.directory = "/opt/kde2/share/wallpapers /usr/share/WindowMaker/Backgrounds $HOME/GNUstep/Library/WindowMaker/Backgrounds";
+        data->param.directory.directory = "/usr/mpkg/share/WindowMaker/Backgrounds $HOME/GNUstep/Library/WindowMaker/Backgrounds";
         data->param.directory.stripExt = 1;
 
         smenu = putNewSubmenu(pad, _("Assorted XTerms"));
